defaults/main.yml

```diff
@@ -58,5 +58,5 @@ nginx_compile_time_options_builtin_modules:
   - http_stub_status_module
 
 nginx_compile_time_options_external_modules_paths:
-  - /tmp/headers-more-nginx-module-{{ headers_more_version }}
-  - /tmp/lua-nginx-module-{{ lua_module_version }}
+  - /nginxbuild/headers-more-nginx-module-{{ headers_more_version }}
```

Comment:
[FSH (Filesystem Hierarchy Standard)](http://www.pathname.com/fhs/pub/fhs-2.3.html#PURPOSE18) suggests to put those source files under _/usr/src_